docs/tutorials/near-indexer.md

```diff
@@ -45,10 +45,10 @@ Inside this folder you will find:
 
 Next, you will need to create a Rust toolchain that mirrors the one in [`nearcore`](https://github.com/near/nearcore/blob/master/rust-toolchain.toml):
 
-To do this, run the following command in the root of your project: _(be sure to check the link above for the correct version)_
+To do this, run the following command in the root of your project. This will specify which toolchain your rust environment will use : _(be sure to check the link above for the correct version)_
```

Comment:
```suggestion
To do this, run the following command in the root of your project. This will specify which toolchain your Rust environment will use: _(be sure to check the link above for the correct version)_
```